tools/sof-tplgreader.py

```diff
@@ -183,7 +183,7 @@ def _filterOutput(self, pipelines, filter_dict, bIn):
                     check = True if key in line.keys() else False
                 else:
                     # check if current pipeline is among the ones we want
-                    check = str(line[key]) in value
+                    check = any([em in str(line[key]) for em in value])
```

Comment:
Looks like we're going from an exact match to substring match without a single comment, documentation or `--help` change. People should be able to use these filters without having to find line number 186 and know enough Python to know  what `str1 in str2` does.

Reply:
updated